bus/bus.go

```diff
@@ -580,8 +612,85 @@ func (b *bus) objectsMarkSlabMigrationFailureHandlerPOST(jc jape.Context) {
 	}
 }
 
+func (b *bus) settingsHandlerGET(jc jape.Context) {
+	settings, err := b.ss.Settings()
+	if jc.Check("couldn't load settings", err) != nil {
+		return
+	}
+	jc.Encode(settings)
+}
+
+func (b *bus) settingsRedundancyHandlerGET(jc jape.Context) {
```

Comment:
I don't think you need this handler if you have a generic endpoint to get a setting and update a setting. I left an example in the client file further down.

Reply:
resolving in favour of https://github.com/SiaFoundation/renterd/pull/55/files#r1043481421